backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/material/cockpit/rowfactory/HighPriceProvider.java

```diff
@@ -76,6 +77,16 @@ private Map<HighPriceRequest, HighPriceResponse> computeHighestPrices(final Set<
 		for (final HighPriceRequest request : requests)
 		{
 			final ProductId productId = ProductId.ofRepoId(request.getProductDescriptor().getProductId());
+			final I_M_Product productRecord = productBL.getById(productId);
```

Comment:
don't we have IProductBL.getByIds(Set<ProductId>) ?

Reply:
not on this branch yet